services/drupal/web/themes/epa_theme/source/_patterns/05-components/external-link/_external-link.scss

```diff
@@ -22,11 +22,21 @@
   }
 }
 
-a#{$external-urls} {
-  @extend .external-link;
+html.no-js {
```

Comment:
@jbwagner This generates the following error in Stylelint: `Unexpected qualifying type selector   selector-no-qualifying-type`

[Stylelint](https://stylelint.io/) is used to lint Sass files. Warnings will break the build, so if you have a valid reason to break Stylelint rules you can have it ignore code in two ways:

1. Add `/* stylelint-disable-next-line */` to the line just before where the Stylelint warning is triggered.

2. To ignore several lines, add `/* stylelint-disable */` before the code in question and add `/* stylelint-enable */` afterwards.

The best way to fix the error here is to do the following:
```
/* stylelint-disable-next-line selector-no-qualifying-type */
html.no-js {
```